docs/configuration/integrations/integrations-next/app-o11y-exporter.md

```diff
@@ -0,0 +1,58 @@
++++
+title = "app_o11y_config"
++++
+
+# app_o11y_config
+
+The `app_o11y_config` block configures the `app-o11y-exporter`
+integration, which is a data exporter for logs/metrics/traces/exceptions
+received by the respectful JavaScript agent
+
+Full reference of options:
+
+```yaml
+  # Enables the app_o11y_config integration, allowing the Agent to automatically
+  # collect metrics,logs and exceptions from the grafana frontend agent
+  [enabled: <boolean> | default = false]
```

Comment:
We've moved away from integrations-next integrations having an enabled field. Now, we treat them as enabled if they're defined in the YAML.